Scaffolding.sln

```diff
@@ -31,23 +31,23 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.Web.CodeGenerators.Mvc", "src\Microsoft.VisualStudio.Web.CodeGenerators.Mvc\Microsoft.VisualStudio.Web.CodeGenerators.Mvc.csproj", "{3E99E653-6B31-45CF-9B3A-30EEE497BE13}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.Web.CodeGenerators.Mvc", "src\VS.Web.CG.Mvc\VS.Web.CG.Mvc.csproj", "{3E99E653-6B31-45CF-9B3A-30EEE497BE13}"
```

Comment:
I figured it would make it easier to know what these were for to have VS in the folder name at least, but if we think it matters I can trim that too.